### ISSUE TITLE
Accessibility: Improve screen reader experience

### ISSUE THREAD

**Issue by mteguh81** [mteguh81/teguh-event-talks-app#4]
To improve accessibility, ARIA attributes should be added to the application. For example, using 'aria-live' on the schedule container to announce updates to screen reader users when the talks are filtered.

**Comment by manankharwar** [comment]
For ARIA attribute testing and screen reader announcements, AccessOps can scan for missing aria-live and other ARIA issues automatically. Chrome extension that identifies exactly which elements need ARIA support. Free for small projects!
